src/Config/RuntimeConfigLoader.cs

```diff
@@ -230,6 +230,15 @@ private static string GetOverriddenName(string fileName)
         return $"{fileName}.overrides{CONFIG_EXTENSION}";
     }
 
+    /// <summary>
+    /// Generates the name of the file based on environment value.
+    /// NOTE: Input File name should not contain extension
```

Comment:
What if it contains the file extension name? How would the end user know NOT to provide the extension? 

Can we be flexible? Check for json extension at the end, if not present add it?

Reply:
This method is not invoked by user. It's an internal method which is invoked when a default scenario is triggered.
The comment is intended for developers who would may want to reuse this method for other purposes.